openapi.yaml

```diff
@@ -208,10 +441,348 @@ definitions:
             type: string
             format: url
             description: A link to further information about the error
-            example: https://developer.oregonstate.edu/documentation/error-reference#1234
+            example: https://developer.oregonstate.edu/documentation/Error-reference#1234
   Errors:
     properties:
       errors:
         type: array
         items:
           $ref: '#/definitions/Error'
+  GameId:
+    type: string
+    description: A unique ID of a poker game
+  GameIds:
+    type: array
+    items:
+      $ref: '#/definitions/GameId'
+  GamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/GameId'
+          type:
+            $ref: '#/definitions/GameType'
+          attributes:
+            type: object
+            properties:
+              round:
+                $ref: '#/definitions/Round'
+              playerIds:
+                $ref: '#/definitions/PlayerIds'
+              minimumBet:
+                $ref: '#/definitions/MinimumBet'
+              maximumBet:
+                $ref: '#/definitions/MaximumBet'
+              betPool:
+                $ref: '#/definitions/BetPool'
+              tableCards:
+                $ref: '#/definitions/Cards'
+            additionalProperties: false
+        required: 
+          - type
+          - id
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  GamePostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/GameType'
+          attributes:
+            type: object
+            properties:
+              playerIds:
```

Comment:
I think when creating a new game, you'll want to specify a list of `memberIds` instead